src/main/java/spleetwaise/address/model/AddressBookModel.java

```diff
@@ -96,4 +97,9 @@ public interface AddressBookModel {
      * Searches for a person with the given Phone
      */
     Optional<Person> getPersonByPhone(Phone phone);
+
+    /**
+     * Searches for a person with the given index
```

Comment:
The JavaDoc comment here could be clearer in specifying which list the index is based on.